src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12px;
+}
+
+header {
+  margin: 10px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+header > a {
+  width: 30%;
+  text-align: center;
+}
+
+a.is-active {
+  color: #00acdc;
+  text-decoration: underline;
+  position: relative;
+}
+
+.is-active::after {
+  content:"";
+  position: absolute;
+  top: 30px;
+  left: 0;
+  width: 37px;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+}
+
+nav {
+  height: 60px;
+  background-color: #fff;
+  width: 70%;
+}
```

Comment:
You have fixed the width of the `nav` class. According to the checklist, it's better not to fix container size unless there is a specific requirement. Let the content size dictate it to avoid overflow or accidental scroll bar.